package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oauth2-mock-server",
-  "version": "3.2.0",
+  "version": "3.3.0",
```

Comment:
@AndTem Could you please rollback this ? Version bumps are usually done by core members.

Reply:
@nulltoken fixed